test_e2e/conftest.py

```diff
@@ -0,0 +1,51 @@
+import os
+from threading import Thread
+from selenium import webdriver
+
+import pytest
+from dotenv import load_dotenv, find_dotenv
+from selenium.webdriver.firefox.options import Options
+
+from todo_app import app
+
+
+from todo_app.data.trello_items import _make_authorised_request
+
+
+def create_trello_board():
+    load_dotenv(find_dotenv(".env"), override=True)
```

Comment:
I'd be tempted to pull this out of this function (it's potentially quite a surprising side-effect)  and probably leave it in `app_with_temp_board`